theia-java-docker/next.package.json

```diff
@@ -15,20 +15,89 @@
         "@theia/file-search": "next",
         "@theia/getting-started": "next",
         "@theia/git": "next",
-        "@theia/java": "next",
         "@theia/json": "next",
         "@theia/markers": "next",
         "@theia/messages": "next",
         "@theia/navigator": "next",
         "@theia/outline-view": "next",
         "@theia/preferences": "next",
+        "@theia/plugin": "next",
+        "@theia/plugin-ext": "next",
+        "@theia/plugin-ext-vscode": "next",
         "@theia/search-in-workspace": "next",
         "@theia/terminal": "next",
-        "@theia/textmate-grammars": "next",
-        "@theia/tslint": "next",
-        "@theia/typescript": "next",
         "typescript": "latest"
```

Comment:
needed?

Reply:
> needed?

I wasn't sure, it looks like all images include `typescript` as a dependency.